### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by rapka**
These are various minor changes I originally included in https://github.com/icosa-gallery/open-brush/pull/136.

### CODE REVIEW

File: Assets/Editor/BrushLister.cs [hunk 1]
```diff
@@ -36,7 +36,9 @@ static void ListBrushes()
         foreach (BrushDescriptor brush in experimentalBrushes)
         {
             try { AppendValidBrushString(brush, true); }
-            catch (Exception UnassignedReferenceException) { }
+            catch (Exception UnassignedReferenceException) {
```
Comment (rapka) [inline Assets/Editor/BrushLister.cs:39]: This was throwing an unused variable warning and logging these exceptions will probably help debugging in the future.

File: Assets/Editor/BuildTiltBrush.cs [hunk 2]
```diff
@@ -309,7 +309,7 @@ public static TiltBuildOptions GetGuiOptions()
     }
 
     // Menu items
-    [MenuItem("Tilt/Build/Do Build...", false, 2)]
+    [MenuItem("Tilt/Build/Do Build... #&b", false, 2)]
```
Comment (rapka) [inline Assets/Editor/BuildTiltBrush.cs:312]: This adds a Shift+Alt+B shortcut to the build command.

File: Assets/Scripts/App.cs [hunk 3]
```diff
@@ -45,11 +45,6 @@ public class App : MonoBehaviour
         public const float METERS_TO_UNITS = 10f;
         public const float UNITS_TO_METERS = .1f;
 
-        // These control the naming of various things related to the app. If you distribute your own
-        // build, you need to call is something other that 'Tilt Brush', as that if a Google trademark -
-        // see the BRANDING file for details.
-        // As a minimum, you should change kAppDisplayName.
```
Comment (rapka) [inline Assets/Scripts/App.cs:51]: This is done already.

**Comment by mikeage** [comment]
Can I ask you to please run `pre-commit run -a` once manually? If you run `pre-commit install`, it'll automatically do any required formatting prior letting you create the commit, which is a bit cleaner of a workflow.

**Comment by mikeage** [comment]
LGTM; removing the original Google notes from the README is probably a good idea, since we did that part already.

**Review by andybak — APPROVED** [review]
Yeah. This all looks harmless

**Review by mikeage — APPROVED** [review]
ok, I ran it ;-)

**Comment by mikeage** [comment]
Thanks @rapka

**Review by mikeskydev — APPROVED** [review]
LGTM!

**Pull request merged**